src/components/AuthLayout/AuthLayout.styled.js

```diff
@@ -2,7 +2,6 @@ import styled from 'styled-components';
 
 export const MainContainer = styled.main`
   width: 100%;
```

Comment:
You have to come back that change. And you shouldn't to do changes which is not for current task.